Sources/ExyteChat/Theme/ChatTheme.swift

```diff
@@ -44,9 +44,15 @@ public struct ChatTheme {
         images: ChatTheme.Images = .init(),
         style: ChatTheme.Style = .init()
     ) {
-        self.colors = colors
-        self.images = images
         self.style = style
+        self.images = images
+        
+        // if background images have been set then override the mainBG color to be clear
```

Comment:
This functionality is needed as in order for the background images to display correctly we need to set mainBG to .clear